ios/FluentUI/Drawer/DrawerTokens.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// `DrawerTokens` assist to configure drawer apperance via UIKit components.
+class DrawerTokens: MSFTokensBase, ObservableObject {
```

Comment:
Does it need to implement ObservableObject?

Reply:
nope, it's no longer required.